modules/googleapis/0.0.0-20241220-5e258e33/overlay/private/extensions/constants.bzl

```diff
@@ -0,0 +1,44 @@
+# Maps rules used by googleapis to <flavor> if the rule is registered by the googleapis-<flavor>
+# module or to None if the rule is not supported yet (some may never be).
+RULE_TO_FLAVOR = {
```

Comment:
to confirm, this just lists the "recommended" flavor module, right? I could technically use `googleapis-my-own-flavor` to register `cc_proto_library` if I wanted to (ignoring conflicts with other modules' registrations...)?

Reply:
Yes, you could, only requirement is naming them `googleapis-...`.